src/utils/datetime_test.cljs

```diff
@@ -190,76 +193,75 @@
-                                      (some #(= %
-                                                (-
-                                                 (int (get-month-day-int mocked))
-                                                 (int (get-month-day-int curr-time))))
-                                            [1 30 29 27]))
-          form-full-date            #(str (get-hh-mm %)
-                                          " " (string/capitalize (get-week-day %))
-                                          " " (get-month-day-int %)
-                                          " " (string/capitalize (get datetime/months (t/month %))))
-          today-date                #(str (get-hh-mm %) " today")
-          tomorrow-date             #(str (get-hh-mm %) " tomorrow")
-          write-date                #(cond (today? % curr-time)    (today-date %)
-                                           (tomorrow? % curr-time) (tomorrow-date %)
-                                           :else                   (form-full-date %))
-          will-unmute-in-1-hour     (remove-msecs (time-coerce/to-string mock-in-1-hour))
-          will-unmute-in-8-hours    (remove-msecs (time-coerce/to-string mock-in-8-hour))
-          will-unmute-in-15-mins    (remove-msecs (time-coerce/to-string mock-in-15-minutes))
-          will-unmute-in-two-days   (remove-msecs (time-coerce/to-string mock-in-two-days))
-          will-unmute-tomorrow      (remove-msecs (time-coerce/to-string mock-tomorrow))
-          will-unmute-in-three-days (remove-msecs (time-coerce/to-string mock-in-three-days))
-          will-unmute-in-four-days  (remove-msecs (time-coerce/to-string mock-in-four-days))
-          will-unmute-in-five-days  (remove-msecs (time-coerce/to-string mock-in-five-days))
-          will-unmute-in-six-days   (remove-msecs (time-coerce/to-string mock-in-six-days))]
-      (testing "Mute for minutes and hours"
-        (are [arg expected] (= (datetime/format-mute-till arg) expected)
-         will-unmute-in-15-mins (write-date mock-in-15-minutes)
-         will-unmute-in-1-hour  (write-date mock-in-1-hour)
-         will-unmute-in-8-hours (write-date mock-in-8-hour)))
-      (testing "Weekdays"
-        (are [arg expected] (= (datetime/format-mute-till arg) expected)
-         will-unmute-tomorrow      (write-date mock-tomorrow)
-         will-unmute-in-two-days   (write-date mock-in-two-days)
-         will-unmute-in-three-days (write-date mock-in-three-days)
-         will-unmute-in-four-days  (write-date mock-in-four-days)
-         will-unmute-in-five-days  (write-date mock-in-five-days)
-         will-unmute-in-six-days   (write-date mock-in-six-days)))
-      (testing "Until the user turns it back on"
-        (is (= "you turn it back on" (datetime/format-mute-till datetime/go-default-time))))))
+(deftest format-mute-till-test
+    (with-redefs [t/now (constantly (time-coerce/from-long mock-current-time-epoch))]
```

Comment:
@ibrkhalil, I think the proper way is to redef the dynamic var `*ms-fn*` and not the core functions exposed by cljs-time. As we can see, this var is used throughout the cljs-time code https://github.com/andrewmcveigh/cljs-time/blob/master/src/cljs_time/core.cljs#L289. This is also how other tests in `datetime-test` do.

The dynamic var expects a function that returns a number in milliseconds, so you can skip `from-long` and do `(with-redefs [t/*ms-fn* (constantly mock-current-time-epoch)])`